pkg/apis/serving/v1beta1/configuration_validation.go

```diff
@@ -61,3 +64,26 @@ func (cs *ConfigurationStatus) Validate(ctx context.Context) *apis.FieldError {
 func (csf *ConfigurationStatusFields) Validate(ctx context.Context) *apis.FieldError {
 	return nil
 }
+
+// ValidateLabels function validates service labels
+func (c *Configuration) ValidateLabels() *apis.FieldError {
```

Comment:
you can do the same trick as above 
```suggestion
func (c *Configuration) ValidateLabels() (errs *apis.FieldError) {
```

then `var errs...` won't be required.